rust/crd/src/authentication.rs

```diff
@@ -42,6 +42,12 @@ pub struct KafkaAuthentication {
     /// - Which ca.crt to use when validating the provided client certs
     ///
     /// This will override the server TLS settings (if set) in `spec.clusterConfig.tls.serverSecretClass`.
+    ///
+    /// ## Kerberos provider
+    ///
+    /// This affects client connections and also sets internal connections to use TLS for encryption.
```

Comment:
Marker to self: Is this really activating TLS? Where do the certs come from? Is this maybe a copy/paste leftover? :)

Reply:
Fixed when re-working tests e.g. 2e20d5a